alpha/radio.cpp

```diff
@@ -1,25 +1,56 @@
 #include "radio.h"
 #include "hal.h"
+#include <WiFi.h>
+#include <WiFiClientSecure.h>
+#include <ArduinoJson.h>
+#include "opentracev2.h"
+#include "storage.h"
 
 // WiFi functions, call wifi_connect() to connect to "Test" wifi, wifi_disconnect() to disconnect.
 // Prints helpful debug to console upon call. 
 
-#ifndef WIFI_SSID
-#define WIFI_SSID "test"       // Enter your SSID here
-#endif
-#ifndef WIFI_PASS
-#define WIFI_PASS "password"    // Enter your WiFi password here
-#endif
 #define SSID_DISPLAY_COUNT 1
 
-#define WIFI_RETRY_INTERVAL 30000
+#define WIFI_RETRY_INTERVAL 5000
+
+#define TEMPID_BATCH_SIZE 100
+#define HTTPS_PORT 443
+const char* host = "asia-east2-bettr-trace-1.cloudfunctions.net";
+const char* uri_getTempIDs = "/getTempIDs";
+const char* root_ca = \
```

Comment:
can you use F(..) strings so that these are in flash and not RAM ?

Reply:
F doesn't seem to work, I'll use PROGMEM instead